app/components/work_packages/activities_tab/journals/item_component.html.erb

```diff
@@ -30,6 +30,16 @@
                                                     classes: "work-packages-activities-tab-journals-item-component--user-name ellipsis hidden-for-mobile") do
                     truncated_user_name(journal.user)
                   end
+                  if journal.initial?
+                      header_start_container.with_column(
+                        mr: 1,
+                        classes: "work-packages-activities-tab-journals-item-component-details--journal-type hidden-for-mobile"
```

Comment:
🟠 Is this meant to be hidden in mobile?

Reply:
@jonasjabari I'm not sure I understand the reason for the `.hidden-for-mobile` class here

Reply:
Yes, it we don't have enough width for it on mobile